src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -90,7 +90,7 @@ public class MigrateRepoCommand : CommandBase<MigrateRepoCommandArgs, MigrateRep
         };
         public Option<string> AwsBucketName { get; } = new("--aws-bucket-name")
         {
-            Description = "If using AWS, the name of the S3 bucket to upload the BBS archive to (Not required for GHES 3.8.0 and later)."
+            Description = "If using AWS, the name of the S3 bucket to upload the data archives to (Not required for GHES 3.8.0 and later)."
```

Comment:
This isn't particularly related to your (correct!) change, but I guess we might as well fix it in this PR.

```suggestion
            Description = "If using AWS, the name of the S3 bucket to upload the data archives to. Not required if you are migrating from GitHub Enterprise Server 3.8.0 or later."
```

Reply:
addressed in 14c00bd287a7cb724f1f9d391be98c31d22a0144